metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -105,6 +107,20 @@ object UserConfiguration {
         |`src/main/scala:: src/main/java::`. Syntax such as `src/{main,test}::`
         |is not supported.
         |""".stripMargin
+    ),
+    UserConfigurationOption(
+      "bloop-generate-sbt",
```

Comment:
I'm on the fence about this setting since the plugin file is required for "import build" to work. I feel like we should instead have some setting to disable "import build" entirely, for users who prefer to do it manually 🤔 Any thoughts?

Alternatively, we should have an option to use a global sbt-bloop plugin instead of project-local. But I'm not sure that's a good direction.

Reply:
> I'm on the fence about this setting since the plugin file is required for "import build" to work. I feel like we should instead have some setting to disable "import build" entirely, for users who prefer to do it manually thinking Any thoughts?

If someone has a Bloop plugin inside their workspace already and they really don't want to generate the Sbt file, I think we should let them. They would still want to have an automatic import on sbt file changes. And it's a really clear setting that I don't think anyone will click unless they really want to.

>Alternatively, we should have an option to use a global sbt-bloop plugin instead of project-local. But I'm not sure that's a good direction.

That would be just back to the previous behavior and from what I understand it wouldn't play well with offloading.

Overall, I think this would be needed, because we are basically modifying the user's build definition and we should have an option of user telling us that they don't want it.

Reply:
Can we rename the setting to something like “I already have bloop installed”?

Reply:
Changed to `bloop-sbt-already-installed`